modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -703,11 +703,12 @@ function getIssueFields()
             $issueData['watching'] = "Yes";
         }
         $username = $user->getUsername();
+        $comment  = $desc[0]['issueComment'];
         $issueData['commentHistory'] = getComments($issueID);
         $issueData['attachments']    = $attachments;
         $issueData['whoami']         = $username;
         $issueData['othersWatching'] = getWatching($issueID);
-        $issueData['desc']           = $desc[0]['issueComment'] ?? '';
+        $issueData['desc']           = html_entity_decode($comment) ?? '';
```

Comment:
Can you add a comment (to the code) explaining why this should be safe from XSS attacks?

Reply:
@driusan I approved the PR, can you just make sure the comment added is good for you ?